plugins/installer/override/override.php

```diff
@@ -337,19 +336,26 @@ private function saveOverrides($pks)
 		{
 			$insertQuery->clear('values');
 
-			$createdDate = $today->format('y-m-d h:i:s');
+			$tz = new \DateTimeZone(Factory::getApplication()->get('offset'));
+			$date = new Date('now');
+			$date->setTimezone($tz);
+			$createdDate = $date->toSql(true);
```

Comment:
This has to be stored in UTC or not?

Reply:
Why? How do you want to handle then different timezones and DST?

Reply:
@astridx ?

Reply:
>This has to be stored in UTC or not?

If I am right, it is automatically UTC, as UTC is the default time zone in Joomla\CMS\Date\Date (/libraries/src/Date/Date.php).